guides/devboard_as_flasher.rst

```diff
@@ -0,0 +1,64 @@
+Using an ESP devboard as serial flasher
+=======================================
+
+.. _devboard-as-flasher:
+
+If you have a functioning ESP devboard onto which you can flash firmware via USB port,
+it can be also used to flash/communicate via serial to other devices as well.
+We will refer to the devboard with functional USB_UART bridge chip as flasher board for this guide.
+
+Make sure you've read the :doc:`/guides/physical_device_connection` for properly understanding of the functionality of your flasher devboard.
+
+.. figure:: /guides/images/devboard-as-flasher.png
+    :align: center
+    :width: 75.0%
+
+    Connection diagram for an ESP flash target
+
+You need to make the following electrical connections:
+
+- connect both ``EN`` and ``GND`` together in the flasher devboard
+- ``+5.0V`` or ``3V3`` on the flasher devboard to ``VIN`` or ``3V3`` respectively of the target device
+- ``GND``, or ground
+- ``TX`` of flasher devboard to ``TX`` of the target device
+- ``RX`` of flasher devboard to ``RX`` of the target device
+
+Pulling down ``EN`` by connecting it to ``GND`` on the flasher board prevents
+the ESP chip on flasher module from booting and polluting the serial lines.
+
+.. note::
+
+    - On some boards like the AiThinker ESP32Cam, you additionally need to connect ``IO0`` with ``GND``
+      on the target device to force flash/download mode.
+    - Do not connect 3V3 to VIN of the target devices with a 3V3 LDO as it may lead to brownouts.
```

Comment:
```suggestion
    - Do not connect 3V3 to VIN of the target devices with a 3V3 LDO as it may lead to brownouts.
    
Once the connections are made, plug the flasher board into your computer via USB and proceed with flashing the target board via whichever means you intend to use.
```